R/data-importer-configuration.R

```diff
@@ -70,7 +70,7 @@ DataImporterConfiguration <- R6::R6Class(
           return(NULL)
         }
         dimension <- rClr::clrGet(mappedColumn, "Dimension")
-        return(rClr::clrGet(dimension, "DisplayName"))
+        return(ospsuite.utils::ifNotNull(dimension, rClr::clrGet(dimension, "DisplayName")))
```

Comment:
@hannaei Is there a use case where dimension can be null? Let's discuss this.

Reply:
E.g. when the wrong column is matched as the measurement column (like in the example I mentioned above in my first comment) and therefore no unit can be retrieved, dimension will be null.